ErrorHandling/src/backend/HttpApiClient.ts

```diff
@@ -0,0 +1,29 @@
+import axios, {AxiosInstance, AxiosPromise, CancelToken} from 'axios';
+import {Platform} from 'react-native';
+
+const baseHost = Platform.select({
+  default: 'localhost',
+  android: '10.0.2.2',
+});
+
+const defaultInstance = axios.create({
+  baseURL: `http://${baseHost}:3000`,
+});
+
+export class HttpApiClient {
+  private readonly client: AxiosInstance;
+  constructor(client?: AxiosInstance) {
+    this.client = client ?? defaultInstance;
+  }
```

Comment:
なんか自信なくなってきたんだけど、これじゃだめなんだっけ？？

```suggestion
  constructor(private readonly client = defaultInstance) {}
```

Reply:
それでいいわ

d214ad8c7e4eed4a02451149c0430a9253c7a5b1